src/test/ValidatorKeys_test.cpp

```diff
@@ -136,60 +135,121 @@ class ValidatorKeys_test : public beast::unit_test::suite
             auto const keyType = KeyType::ed25519;
```

Comment:
It feels like we mostly, but don't quite, support secp256k1 keys.  The key type is stored in the `validator-keys.json`, but there's no command line option to set the key type.  I'm good with that state of affairs, if that's where you want to leave it.  But I'm thinking you could omit the `["key_type"]` field from the `validator-keys.json` and raise fewer questions if someone were to peek inside the file.  Your call.